src/app/FakeLib/TypeScript.fs

```diff
@@ -40,8 +40,7 @@ type TypeScriptParams =
       /// Specifies the timeout for the TypeScript compiler.
       TimeOut : TimeSpan }
 
-let private TypeScriptCompilerPath = 
-    @"[ProgramFilesX86]\Microsoft SDKs\TypeScript\1.0\;[ProgramFiles]\Microsoft SDKs\TypeScript\1.0\;[ProgramFilesX86]\Microsoft SDKs\TypeScript\0.9\;[ProgramFiles]\Microsoft SDKs\TypeScript\0.9\"
```

Comment:
Instead of explicitly hard-coding the versions we support by default (1.0, 0.9 currently), the new code discovers what versions the user has installed and uses the latest they have.  This _could_ invisibly upgrade some users to a more recent version of tsc, yes, but if they are dependent on a particular version of tsc we should encourage them to set their tools path.